app/web/src/components/AssetDetailsPanel.vue

```diff
@@ -398,7 +438,7 @@ const _configurableProps = computed(() => {
     };
     config.push(d);
   });
-  config.sort((a, b) => a.name.localeCompare(b.name));
+  config.sort((a, b) => a.path.localeCompare(b.path));
```

Comment:
Subtle, but this fixes the ordering regression unrelated to this work when the props were changed to use full paths recently